cms/static/js/views/course_info_update.js

```diff
@@ -18,31 +19,46 @@ define(['js/views/validation', 'codemirror', 'js/models/course_update',
             initialize: function() {
                 this.template = this.loadTemplate('course_info_update');
                 this.render();
-            // when the client refetches the updates as a whole, re-render them
+                // when the client refetches the updates as a whole, re-render them
                 this.listenTo(this.collection, 'reset', this.render);
                 this.listenTo(this.collection, 'invalid', this.handleValidationError);
             },
 
-            render: function() {
-            // iterate over updates and create views for each using the template
+            resetUpdatesListAndRender: function() {
+                // This method should be used when the course update list needs to be
+                // cleared and then rendered.
                 var updateEle = this.$el.find('#course-update-list');
-            // remove and then add all children
                 $(updateEle).empty();
-                var self = this;
+                this.render();
+            },
+
+            render: function() {
+                // iterate over updates and create views for each using the template
+                var updateEle = this.$el.find('#course-update-list'),
```

Comment:
Nit: don't use abbreviations, and name all JQuery variables starting with ``$``. I'd call this ``$updateList``.